### PR TITLE
[SYCLomatic] Support migration of the macro __CUDA_ALIGN__

### PR DESCRIPTION
**Pull request by wenhuiNi**
Signed-off-by: Ni, Wenhui <wenhui.ni@intel.com>

### CODE REVIEW

File: clang/test/dpct/replace-align.cu [hunk 1]
```diff
@@ -1,6 +1,9 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
 // RUN: dpct --format-range=none -out-root %T/replace-align %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
 // RUN: FileCheck %s --match-full-lines --input-file %T/replace-align/replace-align.dp.cpp
 
+#include <cuda_fp8.h>
```
Comment (tangjj11) [inline clang/test/dpct/replace-align.cu:6]: why need add this?

**Review by tangjj11 — APPROVED** [review]

File: clang/test/dpct/replace-align.cu (hunk 1 above)
Comment (wenhuiNi) [reply clang/test/dpct/replace-align.cu:6]: __CUDA_ALIGN__ is defined in this file.

**Review by yihwang-nv — APPROVED** [review]
LGTM

File: clang/test/dpct/replace-align.cu (hunk 1 above)
Comment (tomflinda) [inline clang/test/dpct/replace-align.cu:6]: cuda_fp8.h is only added in cuda-11.8 or later, so line 1 and line 2 also need to be adjusted.

**Review by ziranzha — APPROVED** [review]

**Review by tomflinda — APPROVED** [review]
LGTM

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**